src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ITypeSymbolExtensions.TypeSyntaxGeneratorVisitor.cs

```diff
@@ -188,6 +188,24 @@ private TypeSyntax TryCreateSpecializedNamedTypeSyntax(INamedTypeSymbol symbol)
                     return CreateTupleTypeSyntax(symbol);
                 }
 
+#if !CODE_STYLE // TODO: Remove the #if once IsNativeIntegerType is available.
```

Comment:
:memo: there is a similar region below for tests involving new nullability APIs

Reply:
note: given the redundancy in checks in these places, IDE will likely just have an extension of IsNint and IsNuint since really, that's what we want to ask :)